spec/tags/truffle/pre_initialization_tags.txt

```diff
@@ -4,3 +4,4 @@ slow:The pre-initialized context is not used when the home is unset but was set
 slow:The pre-initialized context picks up new environment variables
 slow:The pre-initialized context is not used if -Xpreinit=false is passed
 slow:The pre-initialized context is used when $VERBOSE changes
+aot:The pre-initialized context is used when $VERBOSE changes
```

Comment:
This is weird, do you know why it would fail with this PR? It works fine on master afaik.

Reply:
Yeah it's strange I could not figure it out, I've decided to postpone it so I can hopefully get this merged in time for 33

Reply:
Looks like it passes now.
Maybe you just had RUBYOPT or TRUFFLERUBYOPT set in your environment?
That makes it fail if there is `-w` or `-W` in the env var.

Reply:
https://github.com/oracle/truffleruby/pull/1307